package.json

```diff
@@ -40,7 +40,7 @@
       "bundle exec rubocop --require rubocop-rspec --auto-correct"
     ],
     "scripts/{release,stage_release}": [
-      "bundle exec rubocop --require rubocop-rspec --auto-correct --except Style/TopLevelMethodDefinition,Rails/Date"
+      "bundle exec rubocop --require rubocop-rspec --auto-correct"
```

Comment:
:100: this pairs with the changes to .rubocop.yml to handle the existing problem cases without allowing more